src/pywink/test/api_test.py

```diff
@@ -302,12 +332,14 @@ def test_get_camera_updated_states_from_api(self):
         devices = get_cameras()
         old_states = {}
         for device in devices:
-            device.api_interface = self.api_interface
-            device.set_mode("away")
-            device.set_privacy(True)
-            device.update_state()
-        self.assertEqual(device.state(), "away")
-        self.assertTrue(device.private())
+            if isinstance(device, WinkCanaryCamera):
```

Comment:
Instead of a loop and check here, why not just index into the array at the correct position?
The position can be assumed because we have control over the test environment.